src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2475,12 +2510,26 @@ string GetBindingOptions()
 				{
 					return templateBindingNode
 						.Members
-						.Select(m => "{0} = {1}".InvariantCultureFormat(m.Member.Name == "_PositionalParameters" ? "Path" : m.Member.Name, BuildBindingOption(m, FindPropertyType(m.Member), prependCastToType: true)))
+						.Select(BuildMemberPropertyValue)
 						.Concat("RelativeSource = new RelativeSource(RelativeSourceMode.TemplatedParent)")
 						.JoinBy(", ");
 				}
 
 				return null;
+
+				string BuildMemberPropertyValue(XamlMemberDefinition m)
```

Comment:
If the method does not capture any local variables, make it a class-scoped method, not a local one.

Reply:
Done, it's now a private method.